src/wizards/tokenSwapDealWizard/stages/submitStage/submitStage.scss

```diff
@@ -0,0 +1,189 @@
+@import "../../../../resources/elements/primeDesignSystem/_variables.scss";
+@import "../../../../resources/elements/primeDesignSystem/_colors.scss";
+
+@mixin commonContentLabel {
+  margin: 2px 0;
+  font-weight: $font-bold;
+  color: $Secondary02;
+}
+
+@mixin daoAvatar($size: 32px) {
```

Comment:
copied from @paweljanicki 

We might want to generalize this in a common place

Reply:
would it be reasonable to make this a reusable component, used in both places?  Or are there too many differences?

Reply:
Only 2 places.

I wanted to avoid too many unrelated changes in this PR.
But would defo want to refactor in a follow up

Reply:
ok, can you please create a ticket for that?  @hiaux0

Reply:
#287